### ISSUE TITLE
Download does not work

### ISSUE THREAD

**Issue by dtoxic47** [slapperwan/gh4a#857]
Trying to download anything frm release page,starts the download but terminates it immediately,trying again the download goes into loop showing two downloads are started,but nothing is beeing dowloaded, my phone is Fujitsu F-02G, Kitkat 4.4.4

**Comment by dmaulana** [comment]
have you give permission to the app to access storage?
are you able to download repo as a zip ?

**Comment by sick13** [comment]
yes all permissions are granted,downloading as a Zip same thing, it goes into endless loop,Downloading "Untitled", no extension or name of the download is given

**Comment by maniac103** [comment]
This seems to be a problem with Kitkat's download manager not being able to talk to Github anymore:

```
W/DownloadManager( 1759): Aborting request for download 10: javax.net.ssl.SSLHandshakeException: javax.net.ssl.SSLProtocolException: SSL handshake aborted: ssl=0xb7ec0550: Failure in SSL library, usually a protocol error
```

I don't think I can do anything about this :-/

**Comment by Fs00** [comment]
@maniac103 I think this can be closed now, given that #1166 has been merged and therefore the app can't run anymore on devices that don't support TLS 1.2.

**Comment by maniac103** [comment]
Right, thanks 👍
